packages/app-builder/src/routes/ressources+/cases+/add-comment.tsx

```diff
@@ -25,11 +25,13 @@ import { z } from 'zod';
 const MAX_FILE_SIZE_MB = 20;
 const MAX_FILE_SIZE = MAX_FILE_SIZE_MB * 1024 * 1024;
 
-const schema = z.object({
-  caseId: z.string().nonempty(),
-  comment: z.string(),
-  files: z.array(z.instanceof(File)),
-});
+const schema = z
+  .object({
+    caseId: z.string().nonempty(),
```

Comment:
Didn't see that before (I know it's not from your changes), but I think it would be better with a `z.string().uuid()`